app/models/guidance_page.rb

```diff
@@ -0,0 +1,22 @@
+class GuidancePage
+  def initialize(path, content: nil)
+    @path = path
+    @content = content
+  end
+
+  def sub_headings
+    page_contents.scan(/^##\s(.*)$/).map(&:first)
```

Comment:
I wonder if we should have this as `def sub_heading_links` and return a hash here to make rendering the links easier? something like:

```
{
  "#sub-heading-1" => "Sub heading 1",
  "#sub-heading-2" => "Sub heading 2",
}
```

Reply:
Would that lead to the slight difficulty of making sure the `href` matched the `id` generated by the markdown library?

Reply:
Potentially, but I'm not sure if there's a way around that as we render the side navigation outside of the markdown template rendering anyway? one way or another we need to render some side links that link into the separately rendered markdown headings, so the link construction has to happen somewhere?

Reply:
ah I'm forgetting about being able to do `anchor: 'some heading'` in the path :) never mind